spec/services/search/alert_builder_spec.rb

```diff
@@ -9,7 +10,7 @@
   let(:location) { "SW1A 1AA" }
   let(:date_today) { Date.current.to_time }
   let(:location_point_coordinates) { Geocoder::DEFAULT_STUB_COORDINATES }
-  let(:location_radius) { Search::LocationBuilder.convert_radius_in_miles_to_metres(10) }
+  let(:location_radius) { DistanceHelper.convert_miles_to_metres(10) }
```

Comment:
this then changes to `helper.convert_miles_to_metres` or just `convert_miles_to_metres`, I can't remember which